ui/src/settings.ts

```diff
@@ -122,6 +122,14 @@ const PRESETS: Preset[] = [
     declutterEnabled: true,
     equationDiagramsEnabled: true,
   },
+  {
```

Comment:
Now that there are more than a handful of presets, it might be worth adding a comment above each to describe what the preset is and what it's for. That'll help both with readability of this file, and help out who even ends up removing the preset.

Reply:
Done!